paperqa/agents/search.py

```diff
@@ -429,35 +429,36 @@ async def query(
 
 def fetch_kwargs_from_manifest(
     file_location: str, manifest: dict[str, Any], manifest_fallback_location: str
-) -> dict[str, Any]:
-    manifest_entry: DocDetails | None = manifest.get(file_location) or manifest.get(
+) -> dict[Any, Any]:
+    manifest_entry: dict[Any, Any] | None = manifest.get(file_location) or manifest.get(
         manifest_fallback_location
     )
     if manifest_entry:
-        return manifest_entry.model_dump()
+        return DocDetails(**manifest_entry).model_dump()
     return {}
 
 
 async def maybe_get_manifest(
     filename: anyio.Path | None = None,
-) -> dict[str, DocDetails]:
+) -> dict[Any, dict[Any, Any]]:
```

Comment:
I think this should just be `dict[str, dict]` but more preferably `dict[str, dict[str, Any]]`